lxd-migrate/main_migrate.go

```diff
@@ -567,6 +608,16 @@ func (c *cmdMigrate) run(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return fmt.Errorf("Failed to make %s read-only: %w", config.SourcePath, err)
 		}
+
+		// In conversion mode, server expects the volume size hint in the request.
+		if config.InstanceArgs.Source.Type == "conversion" {
+			stat, err := os.Stat(target)
```

Comment:
Does this work for both vm images and source partitions?

See BlockDiskSizeBytes() in LXD

Reply:
Fixed. I've moved `BlockDiskSizeBytes` function into `shared`.